tests/JMS/Serializer/Tests/Serializer/EventDispatcher/Subscriber/DoctrineProxySubscriberTest.php

```diff
@@ -49,6 +49,17 @@ public function testDoesNotRewriteCustomType()
         $this->assertTrue($obj->__isInitialized());
     }
 
+    public function testProxyLoadingCanBeSkippedForVirtualTypes()
```

Comment:
Now we should have a test with `false` behavior. But I really cannot imagine one.